src/view/configuration/configuration.jsx

```diff
@@ -516,34 +539,39 @@ const Configuration = () => {
                             </div>
                           ) : null}
 
-                          {values.instances.length > 1 ? (
-                            <div className="u-gapTop2x">
-                              <ModalTrigger>
-                                <Button
-                                  id="deleteButton"
-                                  label="Delete Instance"
-                                  icon={<Delete />}
-                                  variant="action"
-                                />
-                                <Dialog
-                                  onConfirm={() => {
-                                    arrayHelpers.remove(index);
-                                    setSelectedAccordionIndex(0);
-                                  }}
-                                  title="Resource Usage"
-                                  confirmLabel="OK"
-                                  cancelLabel="Cancel"
-                                >
-                                  Any rule components or data elements using
-                                  this instance will no longer function as
-                                  expected when running on your website. We
-                                  recommend removing these resources before
-                                  publishing your next library. Would you like
-                                  to proceed?
-                                </Dialog>
-                              </ModalTrigger>
-                            </div>
-                          ) : null}
+                          <div className="u-gapTop2x">
+                            <ModalTrigger>
+                              <Button
+                                id="deleteButton"
+                                label="Delete Instance"
+                                icon={<Delete />}
+                                variant="action"
+                                disabled={values.instances.length === 1}
+                              />
+                              {values.instances.length === 1 ? (
+                                <span className="Note u-gapLeft">
+                                  You must have at least one instance to use
+                                  this extension.
+                                </span>
+                              ) : null}
```

Comment:
This delete button and note is also a change unrelated to tooltips. Specifically, we now always show the delete button, but unless there are multiple instances, the button is disabled at the user sees the "You must have at least one instance to use this extension." note.